src/site/content/en/blog/performance-tool-score-diffs/index.md

```diff
@@ -0,0 +1,188 @@
+tags:
+  - post # post is a required tag for the article to show up in the blog.
+  - performance
+---
+
+Load is not a single point in time- there are multiple moments during the load experience that can affect whether a user perceives it as "fast", and if you just focus on one you might miss bad experiences that happen during the rest of the time. **This is why we generally use multiple metrics blended together to create our performance scores;** but different tools use different metrics and different weighted blends. This can be frustrating when you're looking for a source of truth about how fast your website is.
+
+This post serves to explain the reasons that there are differences in performance scores between three Google tools: [Test My Site (TMS)](https://www.thinkwithgoogle.com/intl/en-gb/feature/testmysite), [PageSpeed Insights (PSI)](https://developers.google.com/speed/pagespeed/insights/), and [Lighthouse (LH)](https://developers.google.com/web/tools/lighthouse/). This document does **not** detail other differences in the tools, such as server run locations, network throttling presets, score variance between runs, etc.
+
+## Terminology
+
+Various terms about speed/performance are referenced in our tools. We want to start off with some clarification of how these terms are used.
+
+- **Page speed** refers to the speed of an individual page on your site (URL level).
+- **Site speed** refers to the speed of all of the pages on your site (origin level).
+
+It is important to note that "performance" or "speed" is not to be defined by a single metric (i.e. [First Contentful Paint (FCP)](/first-contentful-paint/), [Time to Interactive (TTI)](/interactive/), etc.). This is motivated by several considerations, including:
+- Page load is not a single moment in time - it's an experience that no one metric can fully capture. [source](https://developers.google.com/web/fundamentals/performance/user-centric-performance-metrics)
+- No single metric in isolation is able to give a meaningful indication of the health of the web (critical for benchmarking).
+- Depending on which metric is chosen, a very different picture of page speed is presented (each requiring different optimizations to improve).
+
+## Performance Scores within tools
+
+The performance score (top level gauge, 1-100 score) in PSI and Lighthouse is calculated from a weighted blend of multiple metrics to summarize a website's/web app's speed. This [weighted blend](https://docs.google.com/spreadsheets/d/1sH_T4G_RZAg4CpcV1bT-tmUegBdBpCOOwsdzqtWnO4U/edit#gid=0) of metrics that together constitute the performance score in both PSI and LH (as of May 2019, [LH v5.0](https://github.com/GoogleChrome/lighthouse/releases)) is as follows:
```

Comment:
So I appreciate what you are trying to do here, but it's still a little vague to me. We can see the weights here, but may not fully comprehend the scoring for each item. Also, what does it mean to be 0X for MPFID?